addons/hr/views/hr_employee_views.xml

```diff
@@ -135,55 +118,72 @@
-                                    <group string="Emergency">
-                                        <field name="emergency_contact"/>
-                                        <field name="emergency_phone" class="o_force_ltr"/>
+                                    <group>
+                                        <group string="Family Status" colspan="2">
+                                            <field name="marital"/>
+                                            <field name="spouse_complete_name" attrs="{'invisible': [('marital', 'not in', ['married', 'cohabitant'])]}" groups="hr.group_hr_user"/>
+                                            <field name="spouse_birthdate" attrs="{'invisible': [('marital', 'not in', ['married', 'cohabitant'])]}" groups="hr.group_hr_user"/>
+                                            <field name="children"/>
+                                        </group>
+                                        <group string="Emergency" colspan="2">
+                                            <field name="emergency_contact"/>
+                                            <field name="emergency_phone" class="o_force_ltr"/>
+                                        </group>
                                     </group>
-                                    <group string="Work Permit" name="has_work_permit">
-                                        <field name="visa_no"/>
-                                        <field name="permit_no"/>
-                                        <field name="visa_expire"/>
-                                        <field name="work_permit_expiration_date"/>
-                                        <field name="has_work_permit"/>
+                                    <group>
+                                        <group string="Education" colspan="2">
+                                            <field name="certificate"/>
+                                            <field name="study_field"/>
+                                            <field name="study_school"/>
+                                        </group>
+                                        <group string="Work Permit" name="has_work_permit" colspan="2">
+                                            <group colspan="2">
+                                                <field name="visa_no"/>
+                                                <field name="permit_no"/>
+                                            </group>
+                                            <group col="4" colspan="2">
+                                                <field name="visa_expire"/>
+                                                <field name="work_permit_expiration_date"/>
+                                            </group>
+                                            <group>
+                                                <field name="has_work_permit" widget="work_permit_upload"/>
+                                            </group>
```

Comment:
I'd make this field take the whole half-column, it looks kinda weird

![image](https://user-images.githubusercontent.com/50135801/156378327-691123ae-1316-4646-bf08-3208018059a7.png)

Reply:
Good catch, thanks!